app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -68,6 +70,8 @@ const messages = defineMessages({
     userId: userIdSelector(state),
     activeProject: activeProjectSelector(state),
     filters: filtersSelector(state),
+    activeFilter: activeFilterSelector(state),
+    activeDashboard: activeDashboardItemSelector(state),
```

Comment:
Not necessary. You can check current page (`launches` or `dashboards` just by `activeLaunchFilter`)